### ISSUE TITLE
ImportError: No module named 'autokey.gtkui.__main__'

### ISSUE THREAD

**Issue by lindeloev** [autokey/autokey#59]
Installing `autokey-gtk` from the ppa on Ubuntu 16.04 works fine. But running fails with the following error:

```
Traceback (most recent call last):
  File "/usr/bin/autokey-gtk", line 9, in <module>
    load_entry_point('autokey==0.93.7', 'console_scripts', 'autokey-gtk')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 542, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2569, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2229, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2235, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named 'autokey.gtkui.__main__'
```

Installing from pip also fails but during install in a way that seems unrelated; see [issue here](https://github.com/autokey-py3/autokey/issues/58). Autokey used to work on this system (I have not re-installed or upgraded Ubuntu or made any risky tweaking of system files), but I don't recall whether I originally installed via `apt-get` or `pip`.

Thanks!!

**Comment by troxor** [comment]
Please remove any existing installation, reinstall from PPA, and try again.

**Comment by lindeloev** [comment]
I had tried `sudo apt-get purge autokey*` and then `sudo apt-get install autokey-gtk` but got the same error. But I fixed the problem by searching my home directory (including hidden folders) for autokey and discovered a lot of config files etc. remaining even after the purge. Purging, removing remaining files (except for my existing scripts) and then installing fixed the problem. So I'm happy and I'm closing this issue.

A minor note: it did not "see" my old scripts on first start. Closing and re-opening fixed it - now everything is as before, except that it added some example scripts to the script pool - I had deleted those. No big problem.
